### DISCUSSION TITLE
How to remove the bounding rectangles ON DRAW EFFECT while using the Brush tool?

### DISCUSSION THREAD

**Discussion by zori** [PintaProject/Pinta#902] [Q&A]
I cannot find a setting in the menu to do that. I am using Pinta to create educational videos, so mostly I am free-hand writing (using a graphical tablet) 
with the brush tool and capturing the screen. The thin transparent bounding rectangles around segments of the brush strokes are VERY OBVIOUS on the video.
<img width="843" alt="Screenshot 2024-07-02 at 16 25 41" src="https://github.com/PintaProject/Pinta/assets/583139/ec477850-5e6e-4d0b-bd8b-8d816bf85024">

I have Pinta 2.1.2 on a MacOS 12.7.5 (Monterey). 
 Sorry if there is an easy fix that I missed. Thanks!

**Comment by cameronwhite** [comment]
I don't think that's intended, but offhand I'm not sure what would cause that...
You could try installing the latest Pinta 2.2 development build from https://github.com/PintaProject/Pinta/releases/tag/dev-preview-builds to see if that behaviour still exists there.

**Reply by zori** [reply]
Ok, I just tried that version. No bounding rectangles, but there seems to be an issue with the brush and pencil not stopping to draw unless you click the touchpad / tap the pencil down (that's not natural gesture on a graphical tablet), which I hadn't noticed in the previous version.

**Reply by cameronwhite** [reply]
Could you file a bug for that please? I don't have a tablet myself to test on, but perhaps something changed with the port to GTK4
